user-interface/6001-SORD-submit_orders.md

```diff
@@ -0,0 +1,139 @@
+  - Good till time `GTT` - not applicable to Market orders <a name="6001-SORD-024" href="#6001-SORD-024">6001-SORD-024</a>
+  - Fill or kill `FOC` <a name="6001-SORD-025" href="#6001-SORD-025">6001-SORD-025</a>
+  - Immediate or cancel `IOC` <a name="6001-SORD-026" href="#6001-SORD-026">6001-SORD-026</a>
+  - Good for normal trading only `GFN` - not applicable to Market orders <a name="6001-SORD-027" href="#6001-SORD-027">6001-SORD-027</a>
+  - Good for auction only `GFA` - not applicable to Market orders <a name="6001-SORD-028" href="#6001-SORD-028">6001-SORD-028</a>
+- **should** only be warned if the time in force is not applicable to the order type I have selected <a name="6001-SORD-029" href="#6001-SORD-029">6001-SORD-029</a>
+- if the user has not set a preference: market orders **should** default to `IOC` <a name="6001-SORD-030" href="#6001-SORD-030">6001-SORD-030</a>
+- if the user has not set a preference: limit orders **should** default to `GTC` <a name="6001-SORD-031" href="#6001-SORD-031">6001-SORD-031</a>
+
+... so I can control if and how my order stays on the order book
+
+## Auto Populating a deal ticket non-manual methods
+
+- TODO Populate by selecting a size/price in the order book
+- TODO Populate by selecting a size/price in the chart
+- TODO Populate by selecting a size/price in the depth chart
+- TODO Input price as a % of account, given the current price field
+
+## See the potential consequences of an order before it is submit
+... based on the current inputs I'd like an indication of the consequences of my order based on my position and the state of the market, I...
+
+- **could** see my resulting open volume <a name="6001-SORD-032" href="#6001-SORD-032">6001-SORD-032</a>
+- **could** see new volume weighted average entry price <a name="6001-SORD-033" href="#6001-SORD-033">6001-SORD-033</a>
+- **could** see a new liquidation level <a name="6001-SORD-034" href="#6001-SORD-034">6001-SORD-034</a>
+- **could** see an estimate of the fees that will be paid (if any) <a name="6001-SORD-035" href="#6001-SORD-035">6001-SORD-035</a>
+- **could** see my "position leverage" TODO - define this
+- **could** see my "account leverage" TODO - define this 
+- **could** see an amount of realized Profit / Loss <a name="6001-SORD-036" href="#6001-SORD-036">6001-SORD-036</a>
+- **could** see any change in margin requirements (if more or less margin will be required) <a name="6001-SORD-037" href="#6001-SORD-037">6001-SORD-037</a>
+- **could** see the notional value of my order <a name="6001-SORD-038" href="#6001-SORD-038">6001-SORD-038</a>
+
+... so that I can adjust my inputs before submitting
+
+## Submit an order
+
+... need to submit my order, when submitting my order, I... 
+
+- **must** submit the [Vega submit order transaction](0013-WTXN-submit_vega_transaction.md). <a name="6001-SORD-039" href="#6001-SORD-039">6001-SORD-039</a>
```

Comment:
Would like to be able to prepare, and optionally sign, but not submit a transaction

Reply:
That's interesting. what about if Console assumed you wanted to broadcast the transaction, and it was left the wallet to handle the scenario where you might not want to broadcast it yet?

Reply:
That's probably fine

Reply:
https://github.com/vegaprotocol/specs/pull/1181/commits/c7ffaa9f56ff18b681f6256d26402751b6852f4f